pandas/tests/scalar/timedelta/test_timedelta.py

```diff
@@ -540,13 +540,11 @@ def test_nat_converters(self):
                 "microsecond",
                 "micro",
                 "micros",
-                "u",
```

Comment:
rather than removing these, could we check the warnings? so, to add something like
```python
if (unit, np_unit) in (('u', 'us'), ('U', 'us'), ('n', 'ns'), ('N', 'ns')):
    warn = FutureWarning
else:
    warn = None
with tm.assert_produces_warning(warn, match=<part of deprecation message goes here>):
    <code which may throw warning goes here>
```

Not unusually a bit fan of `if/then` in tests, but sometimes (like here) it's probably the simplest option